workbench-core/authentication/src/plugins/cognitoAuthenticationPlugin.ts

```diff
@@ -29,6 +29,18 @@ interface TokensExpiration {
   refreshToken: number; // ms
 }
 
+export interface CognitoWebUiClient {
```

Comment:
Rather than setting up a separate client, could we just allow the end user to decide if they want to use the web ui client vs. programmatic client, they can pass those in when setting up the plugin?